### PR TITLE
Update README for new target names

### PR DESCRIPTION
**Pull request by Jumhyn**

### CODE REVIEW

**Comment by CodaFi** [comment]
@swift-ci smoke test and merge

File: README.md [hunk 1]
```diff
@@ -241,7 +241,7 @@ various products. These incremental builds are a big timesaver when developing
 and debugging.
 
     cd ${SWIFT_BUILD_DIR}
-    ninja swift
+    ninja compiler
```
Comment (varungandhi-apple) [inline README.md:244]: Hmm, this was the recommendation I heard, but it still does some work on null builds (maybe it is a phony target?). We could get rid of the `swift-frontend` below and use `swift-frontend` here.

Comment (Jumhyn) [reply README.md]: Gotcha! I've updated the branch appropriately.

Comment (varungandhi-apple) [reply README.md]: Thanks!

**Comment by varungandhi-apple** [comment]
@swift-ci smoke test and merge

**Comment by varungandhi-apple** [comment]
Hmm. Why did the smoke tests not run? 😕 

@swift-ci please smoke test and merge

**Pull request merged**